File: SampleIOStestApp/SmokePodsTest/AutomaticTest.m
//
//  AutomaticTest.m
//  SmokePodsTest
//
//  Created by ibob on 04/01/2016.
//  Copyright © 2016 ibob. All rights reserved.
//

#import "AutomaticTest.h"

@implementation AutomaticTest

-(void)run:(NSDictionary *) urlParams{
    [NSException raise:NSInternalInconsistencyException
                format:@"You must override %@ in a subclass", NSStringFromSelector(_cmd)];
}
@end

File: MFBobTests/iOS/AutomationAppIos/app/AutomationAppIos/AutomationAppIos/AutomaticTest.swift
//
//  AutomaticTest.swift
//  AutomationAppIos
//
//  Created by Amitai Madar on 10/12/15.
//  Copyright © 2015 Amitai Madar. All rights reserved.
//

import Foundation

class AutomaticTest:NSObject {
    
    internal  func run(urlParams :[(String, String)]){
        assert(false, "This method must be overriden by the subclass")
    }
}